bluebottle/utils/validators.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, allowed_mimetypes=None, message=None, code=None):
 
     def __call__(self, value):
         try:
+            value.file.seek(0)
```

Comment:
What does this do?

Reply:
i had an issue where I could not upload a file because for some reason the file had already been read before this check. Possibly because the check was performed twice or something. This makes sure that the file is always read from the beginning and the mme type is properly returned.